examples/simple.rs

```diff
@@ -17,11 +17,14 @@ struct Output {
 struct OutputManager;
 
 struct InputManager;
-struct Keyboard;
+struct ExKeyboardHandler;
+
+impl KeyboardHandler for ExKeyboardHandler {
+    fn on_key(&mut self, mut key_event: KeyEvent) {
+        let keys = key_event.input_keys();
+
+        println!("Got key event. Keys: {:?}. Modifiers: {}", keys, key_event.keyboard().get_modifiers());
```

Comment:
If you could use `wlr_log!` here it will be more in-line with the examples from wlroots.

In general, I want to use `wlr_log!` in all places, even in the examples just so it's clear that this library is wlroots stuff.